core/modules/filter/filter.pages.inc

```diff
@@ -257,8 +257,7 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
     '#title' => t('Width'),
     '#title_display' => 'attribute',
     '#type' => 'number',
-    '#default_value' => isset($values['width']) ? $values['width'] : NULL,
-    '#min' => 1,
+    '#default_value' => isset($values['width']) ? $values['width'] : '',
```

Comment:
If the default value is "0" it should just be wiped out and displayed as an empty string. So replace `isset()` with `!empty()`

Reply:
Done.